blockchain/types/chain_hashes.go

```diff
@@ -18,20 +18,32 @@ package types
 
 import (
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/ser/rlp"
 	"math/big"
 )
 
 type ChainHashes struct {
+	Type uint8
```

Comment:
Where can I find information about type numbers?

Reply:
Currently, we have only type '0'. Do we need enum for this case?

Reply:
If this type is going to be extended in the future, implementing an enum on the type will help to understand it.

Reply:
Type is virtually identical to version number, such as 0, 1, and 2. I considered enum, but decided that it would be meaningless since it was the same form as TYPE0 and TYPE1. Do you have any idea about the naming?

Reply:
Oh, is this numbering for the version? I did not know it. 
If this is version numbering, it doesn't have to be managed as an enum.